tests/config/fgcmFitCycleHsc.py

```diff
@@ -1,12 +1,6 @@
 import lsst.fgcmcal as fgcmcal
 
 config.outfileBase = 'TestFgcm'
-# The unused z-band is here to test the case that extra filters
```

Comment:
We're still getting that extra-filter test with `HSC_FILTER_DEFINITIONS.physical_to_band`, right?